### PR TITLE
Blog app - add forms

### PR DESCRIPTION
**Pull request by brainskev**
In this branch, I achieved the following;
- Created a method called current_user in  ApplicationController to make current user data available to all controllers.
  -  It will return the first user from the database.
- Created forms to perform the following functions:
  -  Creates a Post on behalf of the current_user.
  -  Create a comment on behalf of the current_user.
- Allowed Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def comment_param
+    params.require(:comment).permit(:text)
+  end
```
Comment (CollinsTatang) [inline app/controllers/comments_controller.rb:16]: - [ ] Please the comments controller should have a new and create actions as you did in the post controller.

File: app/views/forms/_comments_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+<%= form_with model: [post, comment] do |form| %>
+  <div class="mb-3">
+    <%= form.label :text %>
+    <%= form.text_area :text, class: "form-control" %>
+  </div>
+  <div>
+    <%= form.submit class: "btn btn-outline-secondary" %>
+  </div>
```
Comment (CollinsTatang) [inline app/views/forms/_comments_form.html.erb:9]: - [ ] Please the should be a form in the views/comments/new.html.erb file which accepts input for creating new comments.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @brainskev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @brainskev, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**